syft/frameworks/torch/tensors/interpreters/placeholder.py

```diff
@@ -31,10 +31,12 @@ def instantiate(self, tensor):
         Add a tensor as a child attribute. All operations on the placeholder will be also
         executed on this child tensor.
 
-        We remove wrappers is there are any.
+        We remove wrappers or Placeholders if is there are any.
         """
-        if hasattr(tensor, "child") and tensor.is_wrapper and tensor.child is not None:
+        if isinstance(tensor, PlaceHolder):
             self.child = tensor.child
+        elif hasattr(tensor, "child") and tensor.is_wrapper and tensor.child is not None:
```

Comment:
The if-statement looks a bit weird.
Can `tensor.is_wrapper` be True while `hasattr(tensor, "child")` is False or `tensor.child is None`?

Reply:
I rmed `tensor.is_wrapper`

Reply:
Ok that's the contrary: I only keep `tensor.is_wrapper` 
The pb here is the this flag (is_wrapper) is not 100% reliable. But here it seems to be ok